package.json

```diff
@@ -27,9 +27,11 @@
   "devDependencies": {
     "@types/jest": "^26.0.22",
     "@types/node": "^14.14.37",
+    "@types/react": "^16.14.0",
     "@zerollup/ts-transform-paths": "^1.7.18",
     "jest": "^26.6.3",
-    "prettier": "^2.2.1",
+    "prettier": "^2.3.0",
```

Comment:
Would not it be better to have prettier version fixed (to avoid unexpected updates of source code) ?